packages/atomic/src/components/result-template-components-v1/atomic-result-sections/atomic-result-section-visual.tsx

```diff
@@ -2,19 +2,18 @@ import {Element, Component} from '@stencil/core';
 import {containsVisualElement} from '../../../utils/utils';
 
 /**
- * The visual section helps the information seeker understand information about
- * the item visually, for example in Commerce an image is a great descriptive
- * for some category of products, an icon can help understand what type an item
- * is or an avatar understand whom it is related to.
+ * This section provides visual information about the item.
+ * For example, in Commerce, an image is a great shorthand for a product category.
+ * An icon can quickly show the item type, or an avatar can quickly show the target customer.
  *
  * Behaviour:
- * * Has a fixed size that depends on the specified image size, the layout, the density, and screen size.
- *   * When the image size is set to icon, this section stays very small.
+ * * Has a fixed size that depends on the specified image size, the layout, the density, and the screen size.
+ * ** When the image size is set to `icon`, this section stays very small.
```

Comment:
Similar to above, is this double-asterix an accident?

```suggestion
 * * When the image size is set to `icon`, this section stays very small.
```

Reply:
Ah, good catch. This item is correct, but the following two entries are missing their second astrix.